htpclient/binarydownload.py

```diff
@@ -135,7 +135,7 @@ def check_prince(self):
     

```

Comment:
The check_client_version is not using the new path configs, is this intended?

Reply:
Same goes for prince

Reply:
Discussed this with @s3inlc those call are deprecated. Use preprocessor if you ever run into this issue.